### PR TITLE
Updated bar file

### PR DESCRIPTION
**Pull request by DooMIM**
### Type of Change

- Adding/Editing a new entry

### Checklist

- [✅ ] All writings are my own.
- [✅] My entry follows the Codecademy Docs style guide.
- [✅] My changes generate no new warnings.
- [✅] I have performed a self-review of my own writing and code.
- [✅] I have checked my entry and corrected any misspellings.
- [✅ ] I have made corresponding changes to the documentation if needed.
- [✅ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [✅] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+A **`bar()`** function returns a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.
+
+## Syntax
+
+```plt.bar(x, height, width, bottom, align)```
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:20]: This should be a ```pseudo code block

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Tags:
+  - 'Graphs'
+  - 'Libraries'
+  - 'Matplotlib'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+A **`bar()`** function returns a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:16]:
```suggestion
A **`.bar()`** function returns a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.
```

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+Title: 'bar'
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:2]:
```suggestion
Title: '.bar()'
```

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Description: 'A bar is a chart/graph that represents categorical data with rectangular bars.'
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:3]:
```suggestion
Description: 'Returns a chart/graph that represents categorical data with rectangular bars.'
```

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  - `height`: The height(s) of the bars.
+  - `width`: The width(s) of the bars default: 0.8  - 
+  - `bottom`: The y coordinate(s) of the bars default: 0
+  - `align`: {‘center’, ‘edge’}, optional, default: ‘center’
+
+## Example
+
+Goal: Return the value of "X", "Y" und "Z" in a graph representation. The `x` represnts the x-axis while the `y` represnts the y-axis. In the X-axis, we define the "X", "Y" and "Z" parameters that we want to represent. In the y-axis, we define the value of each parameter. The last piece of code specifies the bar color and shows it.
+
+```import matplotlib.pyplot as plt
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:33]: This needs to be a ```py code block

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@DooMIM I had more comments. Please make the commits to the existing branch, you don't have to create a new PR. The commits will show in this PR. I am closing the prior PR because it's now a duplicate.

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md (hunk 1 above)
Comment (DooMIM) [reply content/matplotlib/concepts/pyplot/terms/bar/bar.md:20]: @SSwiniarski could you give an example? If the code is written in a descriptive language, ex: plt.bar(x coordinates, height of the bars, width of the bars, and so on).

 I will then remove the parameters. Would this be okay or am I not getting what you mean by pseudo code?

Comment (SSwiniarski) [reply content/matplotlib/concepts/pyplot/terms/bar/bar.md:20]: When I say ```pseudo code blocks I am referring to enclosing the content with three backticks followed by the word "pseudo" on its own line, and following with three backtics on its own line. Refer to the [Code](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#code) and ["Supported Code Blocks" ] (https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#supported-code-blocks)part of the style guide.

it should look like this:
```` 
```pseudo
plt.bar(x, height, width, bottom, align)
```
````

**Comment by DooMIM** [comment]
@SSwiniarski I created a new commit and pushed it but can't create a new pull request. Is that the normal procedure? It is visible in the timeline above though, called barFileUpdate2. 
Screenshot 2023-03-08 at 21 00 45

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md (hunk 1 above)
Comment (DooMIM) [reply content/matplotlib/concepts/pyplot/terms/bar/bar.md]: @SSwiniarski perfect! Thanks for the elaboration.

**Comment by SSwiniarski** [comment]
@DooMIM You don't want to create a new pull request. Just commit to this branch so all the edits are on this pull request.

**Review by SSwiniarski — APPROVED** [review]
@DooMIM This should be ready for a second review.

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+A **`.bar()`** function returns a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.
+
+## Syntax
+
+``` pseudo
+plt.bar(x, height, width, bottom, align)
+```
+
+The parameters to the function are:
+  - `x`: The x coordinates of the bars.
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:25]:
```suggestion
  - `x`: Float or array-like values (may be an index and is not required to be scalar). 
```

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+A **`.bar()`** function returns a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.
+
+## Syntax
+
+``` pseudo
+plt.bar(x, height, width, bottom, align)
+```
+
+The parameters to the function are:
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:24]: The language describing these parameters matches or is almost identical to that in that Matplotlib documentation. Please be aware that all content must be original, we do not accept entries that include content copied from other sites.

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+  - `x`: The x coordinates of the bars. 
+  - `height`: The height(s) of the bars.
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:26]:
```suggestion
  - `height`: Float or array-like values designating the heights of the bars.
```

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
+---
+Title: '.bar()'
+Description: 'Returns a chart/graph that represents categorical data with rectangular bars.'
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:3]:
```suggestion
Description: 'Returns a chart/graph that represents categorical data with vertical bars.'
```

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+## Syntax
+
+``` pseudo
+plt.bar(x, height, width, bottom, align)
+```
+
+The parameters to the function are:
+  - `x`: The x coordinates of the bars. 
+  - `height`: The height(s) of the bars.
+  - `width`: The width(s) of the bars default: 0.8  -
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:27]:
```suggestion
  - `width`: Float or array-like values designating the widths of the bars (the default is 0.8). 
```

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+
+``` pseudo
+plt.bar(x, height, width, bottom, align)
+```
+
+The parameters to the function are:
+  - `x`: The x coordinates of the bars. 
+  - `height`: The height(s) of the bars.
+  - `width`: The width(s) of the bars default: 0.8  - 
+  - `bottom`: The y coordinate(s) of the bars default: 0
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:28]:
```suggestion
  - `bottom`: Float or array-like values to designate the y coordinates of the bars (default: 0).
```

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+  - `align`: {‘center’, ‘edge’}, optional, default: ‘center’
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:29]:
```suggestion
  - `align`:  Keyword (‘center’ or ‘edge’) designating the bar alignment (default: ‘center’).
```

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+
+## Example
+
+Goal: Return the value of "X", "Y" und "Z" in a graph representation. The `x` represnts the x-axis while the `y` represnts the y-axis. In the X-axis, we define the "X", "Y" and "Z" parameters that we want to represent. In the y-axis, we define the value of each parameter. The last piece of code specifies the bar color and shows it.
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/bar/bar.md:33]:
```suggestion
Goal: Return the value of "X", "Y" und "Z" in a graph representation. The `x` represnts the x-axis while the `y` represnts the y-axis. In the X-axis, we define the "X", "Y" and "Z" parameters that we want to represent. In the y-axis, we define the value of each parameter. The last piece of code specifies the bar color and displays the graph.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@DooMIM I've finished my review. Please update and tag me when you've finished. Thanks

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+Tags:
+  - 'Graphs'
+  - 'Libraries'
+  - 'Matplotlib'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+A **`.bar()`** function returns a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.
```
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/matplotlib/concepts/pyplot/terms/bar/bar.md:16]: I think it's typically recommended to use the alternative method .barh() for horizontal charts

File: content/matplotlib/concepts/pyplot/terms/bar/bar.md (hunk 7 above)
Comment (DooMIM) [reply content/matplotlib/concepts/pyplot/terms/bar/bar.md:24]: @caupolicandiaz thanks for your rephrasing the parameters. I wasn't really sure how to rephrase them.

**Comment by DooMIM** [comment]
@caupolicandiaz perfect! I updated the bar file according to your review. Looking forward for your feedback.

**Review by caupolicandiaz — APPROVED** [review]
@DooMIM Looks good.

**Comment by SSwiniarski** [comment]
@DooMIM Congrats on the new entry. The live version is here: https://www.codecademy.com/resources/docs/matplotlib/pyplot/bar  You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Then log out and log back into Codecademy using your GitHub login:

image

**Pull request merged**